reana-ui/src/pages/workflowList/WorkflowList.js

```diff
@@ -66,7 +66,7 @@ function Workflows() {
     };
   }, [config.pollingSecs, dispatch, pagination, reanaToken]);
 
-  if (!workflows) {
+  if (!workflows || !config.pollingSecs) {
```

Comment:
Minor suggestions: this logic could be moved to `selectors` just to be more declarative

Reply:
Do you mean something like this?

```js
export const getWorkflows = (state) =>
  !!state.config.pollingSecs && state.workflows.workflows;
```

If yes.. I'm not sure.. for now we're only using getWorkflows in a single place, but if we start using it somewhere else it will always depend on the existence of `pollingSecs`, which I'm not sure if it's correct.

Reply:
Actually my initial thought was to move `!workflows || !config.pollingSecs` to selectors, but probably it's not the right place to keep it. Maybe we can create a new variable in the top of the component, something like `const showWelcomePage = !workflows || !config.pollingSecs;` and use this while rendering. It could increase readability a bit

Reply:
Ah ok, I get your point now. Unfortunately, the polling is making our lives harder and harder. I'll create a variable as you suggest.

Reply:
I've created a new selector called `isConfigLoaded` to be more accurate, instead of just checking `pollingSecs`. Let me know what you think. cc/ @audrium

Reply:
I like how it's implemented now! Just a thought: we could modify `config` reducer to hold a flag `isLoaded` or `isFetched` and use that flag in the `isConfigLoaded` `selector`. What do you think?

Reply:
That's indeed better than comparing objects, updated!

Reply:
LGTM!